src/_guides/language/language-tour.md

```diff
@@ -3323,34 +3204,28 @@ as E, T, S, K, and V.
 
 ### Why use generics?
 
-Because types are optional in Dart 1.x, you never *have* to use generics.
-You might *want* to, though, for the same reason you might want to use
-other types in your code: types (generic or not) let you document and
-annotate your code, making your intent clearer.
-
-{% include optional-types-2.0.html %}
+Generics are often required for type safety, but they have other benefits
+than just allowing your code to run:
```

Comment:
other benefits --> more benefits
(otherwise the rest of the sentence doesn't work)